.gitignore

```diff
@@ -23,7 +23,7 @@ docs/_site/
 
 # Theme files
 /theme/
-!/theme/mocha.dark.json
+!/theme/mocha dark.json
 !/theme/solarized.dark.json
 !/theme/tomorrow.dark.json
-!/theme/sweetlove.json
+!/src/main/resources/themes/sweetlove.json
```

Comment:
I realised your new theme files do not have the `.json` extension. Not sure if that's intentional

Reply:
That is intentional